### ISSUE TITLE
Issue converting bitset_to_string to BitSet class from (v1.0.1 to v2.0.3)

### ISSUE THREAD

**Issue by jithujose** [php/pecl-numbers-bitset#11]
I am having an issue when replacing the `bitset_to_string` function with the `BitSet` class (bitset 2.0.3).
My test code:

``` php
<?php
$redis = get_redis();
$key = "pageviews:homepage:latest";

$redis->del($key);
$redis->setbit($key, 1, 1);
$redis->setbit($key, 3, 1);
$redis->setbit($key, 5, 1);
$redis->setbit($key, 8, 1);
$redis->setbit($key, 9, 1);
$redis->setbit($key, 10, 1);
$redis->setbit($key, 15, 1);
$redis->setbit($key, 150, 1); // issue

$bitset = $redis->get($key);

// $ redis-cli GET pageviews:homepage:latest
// "T\xe1\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02"

$data = false;
if (class_exists('BitSet')) {
    $bs = new BitSet();
    $b = $bs->fromRawValue($bitset);
    $data = $b->__toString();

    // Output
    // 00101010100001110000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000

} else {

    $data = bitset_to_string($bitset);

    // Output:
    // 00101010100001110000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001000000
}

var_dump($data);
exit;

```

As you can see from the code, the `bitset_to_string` function is working correctly returning all set bits but the `BitSet` class omits the bit set at offset 150.

Is my usage of `BitSet` class correct or not? If not, how should I replace the `bitset_to_string` function with the class?

**Comment by jithujose** [comment]
@willfitch Any hints?

**Comment by willfitch** [comment]
Hi @jithujose - didn't get an email for your bug report, so I apologize for that.  Let me look into this today, and I'll update you here.

**Comment by jithujose** [comment]
Thank you @willfitch, I will wait for your update.

**Comment by willfitch** [comment]
@jithujose this issue has been fixed and will be released tonight as 3.0.1 and 2.0.4

**Comment by willfitch** [comment]
@jithujose releases are on pecl.  From here forward, please use bugs.php.net to file reports, please :)
